dash_bio/utils/xyz_reader.py

```diff
@@ -48,7 +48,7 @@ def read_xyz(
             # an example line in a xyz file:
             # C    +0.125    -1.032    +2.000
             r = re.search(
-                r'^([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)',
+                r'^\s*([\w]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s+([\w\.\+\-]+)\s*',
```

Comment:
I get the `\s` to match a whitespace character, but why the `*`, just to clarify? Sorry, I'm not a regex expert and I know conventions change across languages...

Reply:
`*` means 0 or more!

Reply:
Oh, duh, thanks! Then suggesting github.com/plotly/dash-bio/pull/343/files#r280881535.